src/components/Profile/MyPosts/MyPosts.js

```diff
@@ -9,15 +9,15 @@ import AddNewPostForm from './AddNewPostForm';
 
 import style from './MyPosts.module.css';
 
-const MyPosts = (props) => {
+const MyPosts = React.memo((props) => {
   const { t } = useTranslation();
 
   const {
     posts,
     addPost,
   } = props;
 
-  const postsElements = posts.map((post) => {
+  const postsElements = [...posts].reverse().map((post) => {
```

Comment:
Consider storing the reversed array in a separate variable.

Reply:
+